app/routes/docs/history.mdx

```diff
@@ -0,0 +1,48 @@
+---
+handle:
+  breadcrumb: History
+---
+
+import history from '~/img/gcn_history.png'
+
+# History of GCN
+
+<img
+  src={history}
+  width="900"
+  height="250"
+  loading="lazy"
+  alt="Timeline of GCN history"
+/>
+
+The development of GCN began in 1992 as an innovative solution to a major hardware failure aboard the Compton Gamma-ray Observatory ([CGRO](https://heasarc.gsfc.nasa.gov/docs/cgro/cgro.html), 1991-2000). The mission was designed such that data was stored on a spacecraft tape recorder and downloaded several times per day via scheduled contacts with NASA's Tracking and Data Relay ([TDRS](https://www.nasa.gov/directorates/heo/scan/services/networks/tdrs_main/)) satellites, which then relayed the data to a ground station. Less than two years into the CGRO mission the tape recorder failed, threatening the continuation of this Great Observatory. To save CGRO, NASA arranged for continuous, real-time transmission of the science data through the TDRS system.
```

Comment:
Which link for CGRO? https://heasarc.gsfc.nasa.gov/docs/cgro/cgro.html  or https://heasarc.gsfc.nasa.gov/docs/cgro/?

Reply:
The docs/cgro page has more comprehensive information about the instruments and data archives.  I'll update the link.